build.html

```diff
@@ -26,4 +28,30 @@ <h3>Bar Chart</h3>
       </div>
     </div>
   {{/if}}
+  {{#and chartHeightSm chartHeightMd}}
+  <style>
```

Comment:
surely you could have just added a class to the element and have these on the css stylesheet?

@hcarneiro what's your opinion on this

Reply:
@tonytlwu an nvm, I see you had to print some variables here.